data/axo/Custom.css

```diff
@@ -0,0 +1,175 @@
+/*
+❤️ Support - https://www.patreon.com/axorax
```

Comment:
Most people probably aren't going to see this but I'm not sure how I feel about having this here